src/routes/transactions/mappers/transactions-history.mapper.ts

```diff
@@ -21,18 +22,10 @@ import { MultisigTransactionMapper } from '@/routes/transactions/mappers/multisi
 import { TransferMapper } from '@/routes/transactions/mappers/transfers/transfer.mapper';
 import { IConfigurationService } from '@/config/configuration.service.interface';
 
-class TransactionDomainGroup {
-  timestamp!: number;
-  transactions!: (
-    | MultisigTransaction
-    | ModuleTransaction
-    | EthereumTransaction
-    | CreationTransaction
-  )[];
-}
-
 @Injectable()
 export class TransactionsHistoryMapper {
+  private readonly VANITY_ADDRESS_CHARS = 4;
```

Comment:
Nitpick: for the final implementation maybe we can extract the number of chars considered as prefix/suffix to the configuration?